configure.py

```diff
@@ -56,7 +58,10 @@
     "sana": "terminusresearch/sana-1.6b-1024px",
     "ltxvideo": "Lightricks/LTX-Video",
     "wan": "Wan-AI/Wan2.1-T2V-1.3B-Diffusers",
-    "hidream": "HiDream-ai/HiDream-I1-Full"
+    "hidream": "HiDream-ai/HiDream-I1-Full",
+    "auraflow": "terminusresearch/auraflow-v0.3",
```

Comment:
Should this be the official repo? `fal/AuraFlow-v0.3`

Reply:
no way. i can update and fix configs in my own repo. they're non-responsive.